herd/AArch64Annot.ml

```diff
@@ -28,11 +28,11 @@ let is_non_temporal = function
   | _ -> false
 
 let is_atomic = function
-  | XA | XQ | XL | X | NoRet -> true
+  | XA | XQ | XL | X | XNoRet -> true
```

Comment:
Not an objection. I contemplated why we need to flag SWP, CAS when the destination register is ZR and ST<op> as atomics. We would allow execution where there is an intervening write between the Memory Read Effect and the Memory Write Effect of the atomic. Software can't tell the difference and so these executions don't matter but the graph would be very misleading.

For this reason, I think we need `XNoRet` to be atomic.